native/src/routes/language/containers/ChangeLanguageModalContainer.js

```diff
@@ -41,7 +41,7 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
   const availableLanguages: Array<string> = ownProps.route.params.availableLanguages
   const previousKey = ownProps.route.params.previousKey
 
-  const newsRouteMapping = state.cityContent?.newsRouteMapping
+  const newsRouteMapping = state.cityContent?.routeMapping
```

Comment:
Why don't we have to check if it is news-type?

Reply:
Added a check